setup.py

```diff
@@ -18,6 +18,9 @@
     install_requires=[
         'Flask'
     ],
+    extras_require={
+        'raven': ['raven']
```

Comment:
One small nuance that I couldn't decide on:
Do you like the way this is named?

From the docs:
> A dictionary mapping names of “extras” (optional features of your project) to strings or lists of strings specifying what other distributions must be installed to support those features.

The optional feature is Sentry, but raven is the optional package. Maybe this should be
`extras_require={'Sentry': ['raven']}`? Probably doesn't matter all that much.

Reply:
Right now, installing the optional dependency would be like `pip install flask-logger[raven]`, but if I were to update it according to my comment above, it would be `pip install flask-logger[Sentry]`.

Reply:
Ah... the optional feature is Sentry, but they named their package raven which is weird. I kind of like [Sentry] a little better. Any thoughts on why that wouldn't be the best way to go?

Reply:
Changed to the flask-logger[Sentry] option